app/models/offer.rb

```diff
@@ -0,0 +1,4 @@
+class Offer < ApplicationRecord
+  validates :description, presence: true,
+                    length: { minimum: 5 }
```

Comment:
ok, looking good, but we need more validations. Validate presence of other attributes as well.

Reply:
Added more validation